classes/PodsAdmin.php

```diff
@@ -433,49 +434,44 @@ public function admin_menu() {
 						continue;
 					}
 
+					$menu_location        = pods_v( 'menu_location', $pod['options'], 'default' );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 8 spaces

<sub>hash: 74800499574bf36801abc331c697aa90</sub>

Reply:
```suggestion
					$menu_location = pods_v( 'menu_location', $pod['options'], 'default' );
```